app/scripts/services/serviceInstances.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('ServiceInstancesService', function() {
+    return {
+      // NOTE: should this be a filter?  or will shared functionality grow here? 
```

Comment:
Moved the function to a filter, left a comment for a `TODO` on `serviceClassDisplayName` per my previous comment.